README.md

```diff
@@ -27,10 +27,13 @@ kind: OpenTelemetryCollector
 metadata:
   name: simplest
 spec:
+  image: otel/opentelemetry-collector:latest
   config: |
     receivers:
       jaeger:
-
+        protocols:
+          grpc:
+            endpoint: "localhost:14250"
```

Comment:
IIRC the endpoint can be omitted as it is the default. 

Does the operator properly changes objects if a non-default port is being used?

Reply:
Updated to remove the endpoint.

No the service port is not updated - also not sure how this would deal with additional receiver ports - so I'll create a separate issue for this.